Pipfile

```diff
@@ -29,6 +29,8 @@ jinja2 = "*"
 requests = "*"
 importlib-resources = "*"
 conan = "==1.47"
+pytest = "*"
```

Comment:
pytest and mypy are already listed in the `dev-packages` section, I don't think we need them here?

Reply:
I have to admit I'm not super clear on how `Pipfile` works, all I know is that at some point I ran `pipenv lock` which updated `Pipfile` and `Pipfile.lock`. I'll go back and revisit. I'm also not clear on how `conan==1.47` in the (global?)  `Pipfile` interacts with the VFX-year specific version (currently at 1.58).